src/content/en/tools/chrome-devtools/console/command-line-reference.md

```diff
@@ -17,6 +18,10 @@ starting the profiler, and monitoring DOM events.
 Note: This API is only available from within the console itself. You cannot access the Command
 Line API from scripts on the page.
 
+Note: If you are looking for functions that write to the console 
```

Comment:
s/console/Console/

I always capitalize when referring to specific components of DevTools.